frontend/app/src/pages/playground/index.vue

```diff
@@ -1,10 +1,50 @@
 <script setup lang="ts">
-const message = 'put your changes here';
+import { SUPPORTED_MODULES } from '@/types/modules';
+
+const { supportedChains } = useSupportedChains();
```

Comment:
@lukicenturi instead of putting it in the playground doesn't it make sense to add it in the DevApp.vue component?

Reply:
okay moved to the DevApp